grafana/provisioning/dashboards/metrics.json

```diff
@@ -907,11 +905,62 @@
             ],
             "title": "Resources",
             "type": "row"
+        },
+        {
+            "collapsed": true,
+            "gridPos": {
+                "h": 1,
+                "w": 24,
+                "x": 0,
+                "y": 3
+            },
+            "id": 18,
+            "panels": [
+                {
+                    "datasource": {
+                        "type": "loki",
+                        "uid": "${logs_ds}"
+                    },
+                    "gridPos": {
+                        "h": 8,
+                        "w": 24,
+                        "x": 0,
+                        "y": 1
+                    },
+                    "id": 17,
+                    "options": {
+                        "dedupStrategy": "none",
+                        "enableLogDetails": false,
+                        "prettifyLogMessage": false,
+                        "showCommonLabels": false,
+                        "showLabels": true,
+                        "showTime": false,
+                        "sortOrder": "Descending",
+                        "wrapLogMessage": true
+                    },
+                    "pluginVersion": "10.2.0",
+                    "targets": [
+                        {
+                            "datasource": {
+                                "type": "loki",
+                                "uid": "${logs_ds}"
+                            },
+                            "editorMode": "code",
+                            "expr": "{container=~\"backend|jellyfish1|jellyfish2\"}",
```

Comment:
Why something like that? I mean I think I get it why, but I'd create a variable for that and enable multi option - it'd be cleaner and not hardcoded I think